onesignal-public.php

```diff
@@ -306,6 +298,15 @@ public static function onesignal_header()
               <?php
                 }
             }
+        
+            if ($onesignal_wp_settings['prompt_auto_register'] === true) {
+                    echo "OneSignal.showSlidedownPrompt();";
```

Comment:
Just noticed we also have a call to `OneSignal.showHttpPrompt();` above if `use_slidedown_permission_message_for_https` is enabled. However I think we can clean this up now as I don't see us reading the key but please double check to be sure.

Reply:
Cleaned this up